definitions/procedures/backup/metadata.rb

```diff
@@ -37,12 +46,7 @@ def save_metadata(metadata, spinner)
 
     def release_info(spinner)
       spinner.update('Collecting system release info')
-      execute!('cat /etc/redhat-release').chomp
-    end
-
-    def rpms(spinner)
-      spinner.update('Collecting installed RPMs')
-      execute!('rpm -qa').split("\n")
+      "#{os_name} #{os_version_id}"
```

Comment:
Does it make sense to replace this with `cat /etc/os-release` now?

Reply:
Do you mean to collect whole file? If not then `os_name` and `os_version_id` is getting pulled from `/etc/os-release` already?

Reply:
I do indeed mean to collect the entire file. My reasoning is that previously it did `cat /etc/redhat-release` but `/etc/os-release` feels like a more natural replacement. On Ubuntu it also shows the exact patch version (`VERSION="20.04.3 LTS (Focal Fossa)"`) which could be useful, while `VERSION_ID` doesn't contain that.

That said, I don't know for sure where this is exactly used so it may be inappropriate to collect the entire file.

Reply:
Its not getting used as such in restore process, however I agree that instead of VERSION_ID we should collect the VERSION. Adding the method for same in os_facts concern.